src/main/java/com/github/jbellis/jvector/example/Bench.java

```diff
@@ -42,11 +42,6 @@
  * Tests GraphIndexes against vectors from various datasets
  */
 public class Bench {
-
```

Comment:
was this an experiment or did it make a difference?

Reply:
it does help

Reply:
should we move it into SimdOps then?

Reply:
Sure